server/src/test/kotlin/pishello/hello/TestUtilities.kt

```diff
@@ -10,7 +10,7 @@ open class TestUtilities {
     fun correctUserName() = "test_user"
 
     fun correctAuthRequestData(): String =
-        """{ "name": "test_user", "password": "yes, this is a hash" }"""
+        """{ "name": "test_user", "password": "yes, this is a hash", "passwordRepeat": "yes, this is a hash" }"""
```

Comment:
a czy w takim razie tego potrzebujemy?

Reply:
Potrzebujemy do testu _shouldRegisterUser_

Reply:
ale czy jeśli nie przesyłamy na backendzie informacji o repeatedPassword, to czy ten test dalej ma racje bytu?

Reply:
Test był już wcześniej, sprawdza czy udało się zarejestrować użytkownika

Reply:
No tak, ale chodzi o to, że jeśli finalnie nie jest nic zmieniane w serwerze i weryfikacja jest wyłącznie na froncie, to nie potrzeba zmieniać testu bo ten endpoint nigdy takiej danej jak passwordRepeat nie otrzyma. Podobnie nie potrzeba dodawać tych zależności w Gradle'u, a przynajmniej tak mi się wydaje

Reply:
Już poprawione